x-pack/metricbeat/module/prometheus/remote_write/remote_write_test.go

```diff
@@ -1016,7 +1016,7 @@ func TestGenerateEventsHistogramsDifferentLabels(t *testing.T) {
 		},
 		"http_request_bytes": mapstr.M{
 			"histogram": mapstr.M{
-				"values": []float64{float64(0.125), float64(0.375), float64(0.75)},
+				"values": []float64{float64(0.125), float64(0.375), float64(0.5)},
```

Comment:
Trying to understand this change: Why we have now 0.5?

Is it due to // Report +Inf bucket as a point, use the preceding bucket's value, and we put the last bucket?

Reply:
yes, for the last bucket (+Inf 'le') is used the preceding bucket's value, instead of interpolation by adding half the difference between the previous two buckets to the second last bucket